operations/string_processing/indexany/operation.go

```diff
@@ -0,0 +1,37 @@
+package lastindex
```

Comment:
Is this package name correct in indexany?

Reply:
changed